examples/config.yml

```diff
@@ -13,11 +13,11 @@ pki:
   # disconnect_invalid is a toggle to force a client to be disconnected if the certificate is expired or invalid.
   #disconnect_invalid: true
 
-  # default_version controls which certificate version is used in handshakes.
+  # outgoing_version controls which certificate version is used when initiating handshakes.
```

Comment:
Seeing it here now makes me think `initiating_version` might make the most sense?

Reply:
I agree, I think I like initiating version best.

Reply:
I'm fine with initiating. I considered it first, since I know we commonly use that terminology in the code, but I wasn't sure if it was as clear.

The argument for outgoing is that from "my" perspective, I or the other end may be "initiating" the connection, making this config option a little ambiguous. But "outgoing" tunnels are clearly tunnels that originate from my end.

The argument for initiating is that it's language we use everywhere else when talking about standing up tunnel. And "outgoing" has the potential to confuse with outgoing connections vs. outgoing handshakes.

Reply:
Updated the PR to `initiating_version`